v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -80,25 +169,11 @@ const isValid = isValidSubstrateAddress()
 console.log(isValid)
 ```
 
-### Community Built
-
-Below you will find a list of other Substrate API tools built in various languages by our community.
-They each validate addresses in their own way, and can act as a starting point.
-
-- [Python Substrate Interface](https://polkascan.github.io/py-substrate-interface/#substrateinterface.Keypair) - developed by Polkascan.
-
-- [Go Substrate RPC client](https://github.com/centrifuge/go-substrate-rpc-client) - developed by Centrifuge.
-
-- [C++ Polkadot Substrate API](https://github.com/usetech-llc/polkadot_api_cpp) - developed by Usetech.
-
-- [.Net API](https://github.com/usetech-llc/polkadot_api_dotnet) - developed by Usetech.
-
-- [Go implemented utilities](https://github.com/itering/subscan-essentials) - developed by Subscan.
-
-- [SS58 Transform](https://polkadot.subscan.io/tools/ss58_transform) - to manually verify an address (sanity check).
```

Comment:
You want to get rid of the community section?

Reply:
yes, it's moving to the awesome list.

Reply:
Maybe we could link there?